git_setup_windows/git_setup_windows.md

```diff
@@ -0,0 +1,339 @@
+<!--
+
+author:   Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  0.0.1
+module_template_version: 2.0.0
+language: en
+narrator: UK English Female
+title: Setting Up Git on Windows
+
+comment: This module provides recommendations and examples to help new users configure git on their Windows computer for the first time.
+
+long_description: If you're ready to start using the Git version control system, this lesson will walk you through how to get set up. This lesson should be a good fit for people who already have an idea of what version control is but may not have any experience using it yet. No previous experience with Git is expected. This lesson is specific to Windows machines, If you are using Mac or Linux, please follow along with the [set-up guide for those computers]( ).
+
+estimated_time: 25 min
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- Download `git` onto a Windows computer
+- Configure `git` the first time it is used on a computer
+- Understand the meaning of the `--global` configuration flag
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Setting Up Git on Windows
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites**
+
+- Have an account on github.com (you can [sign up now](https://github.com/signup) if you haven't yet --- it's free)
+
+
+**Learning Objectives**
+
+@learning_objectives
+
+</div>
+
+## Downloading Git
+
+Since Windows machines do not come with `git` pre-installed, you will need to download the latest version of [Git for Windows](https://git-scm.com/download/win).
```

Comment:
Ok, it seems like if they have git, but not the Git Bash application, we are going to need them to re-download the whole thing anyway, so I've added a part in the Lesson Preparation section to address that.

Reply:
Sounds good to me. For clarity, it might be worth revising this to "For machines that do not yet have Git Bash installed, you will need to start by downloading the latest version of [Git for WIndows](https://git-scm.com/download/win)." Just in case they skimmed the previous section, or read it but paged to this one rather than skipping directly to using git bash.

Reply:
good call, done!